api-spec/api-spec.md

```diff
@@ -38,22 +38,21 @@ STAC API.
 
 ## HTTP Request Methods and Content Types
 
-The OAFeat and STAC APIs follow a RESTful model. A core principal of this is the use of HTTP Request Methods (verbs) and
-the `Content-Type` header to drive behavior. This section describes how these are used in the OAFeat and STAC endpoints. 
+The OAFeat and STAC APIs follow a RESTful model. A core principal of this is the use of HTTP Request Methods ("verbs") and
+the `Content-Type` header to drive behavior on resources ("nouns"). This section describes how these are used in the OAFeat and STAC endpoints. 
 
-1. **Required** GET (both OAFeat and STAC)
-2. **Recommended** POST `Content-Type: application/x-www-form-urlencoded` with the corresponding content body format.
-3. **Recommended** POST `Content-Type: multipart/form-data` with the corresponding content body format.
-4. **Optional** **STAC endpoint /search only** POST `Content-Type: application/json`, where the content body is a JSON 
+1. **Required** Both OAFeat and STAC: GET using query string parameters
```

Comment:
does this translate to: all GET endpoints are required?

Reply:
Yes. Implementing all OAFeat endpoints and `GET /search` is required be considered "compliant" with STAC API. I took this approach for two reasons: 1) STAC is supposedly an extension of OAFeat, so I didn't see how that could be the case an NOT require the OAFeat endpoints to be implemented 2) We've had the convention elsewhere that **something** has to be required in each extension.